src/translations/en_DK.json

```diff
@@ -251,7 +251,7 @@
   "ONBOARDING_DOWNLOAD_HEADLINE_PART_2": "to get started!",
   "ONBOARDING_DOWNLOAD_HEADLINE_PART_1": "Download Hedvig",
   "LOGIN_APP_HEADING": "Sign in",
-  "LOGIN_APP_BTN_LABEL": "Sign in with BankID",
+  "LOGIN_APP_BTN_LABEL": "Sign in with NemID",
```

Comment:
not sure i like this. Rather, i think we should make a new text key if the meaning is different (it doesn't refer to the same thing)

Reply:
We've discussed this before and I don't agree. With that said I didn't change this, I think Maggy did. I should maybe just have removed it since it's out of scope for this PR but it seemed urgent to get these new links out since we're currently linking to the wrong terms and conditions.

Reply:
Haha, the eternal discussion 😅